### PR TITLE
fix(EMI-1640): Update ask specialist workflow engine config

### PR DESCRIPTION
**Pull request by MrSltun**
The type of this PR is: **Fix**

This PR solves [EMI-1640]

### Description
This PR updates the Ask Specialist inquiry modal config to fix showing the Error status in the button.
~~We also updated the waiting period from `0.5s` to `1.5s` so users don't feel surprised when the modal suddenly disappears~~
We also added a toast when the message gets sent.

#### The motive behind the solution:
The fix here was to remove the `isLoggedOut` check from the `AskSpecialist` config for two reasons:
- We simplified the workflow a while ago in #11399. In the previous workflow, we used to have a confirmation modal at the end of the steps, so if the user was already logged in, it would skip the `isLoggedOut` check and move on to the confirmation modal.
- We don't have an Ask Specialist inquiry flow anymore, and if the user was logged in, the `isLoggedOut` check doesn't have a next step to go to, and it breaks

Due to these reasons, removing the check made sense

### Screenshots & Videos
| Before | After |
|---|---|
| <video src="https://github.com/artsy/force/assets/20655703/5f0f4ef8-3666-4b63-be5b-1ff3b9ad7c2a" /> | <video src="https://github.com/artsy/force/assets/20655703/78e66bfd-f801-491f-a726-6d558696d484" /> |

| New Toast |
|---|
| image |

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EMI-1640]: https://artsyproduct.atlassian.net/browse/EMI-1640?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Components/Inquiry/Views/InquirySpecialist.tsx [hunk 1]
```diff
@@ -59,7 +59,7 @@ export const InquirySpecialist: React.FC = () => {
         contactGallery: false,
       })
       setMode("Success")
-      await wait(500)
+      await wait(1500)
```
Comment (dzucconi) [inline src/Components/Inquiry/Views/InquirySpecialist.tsx:62]: What about just sending a toast?

**Review by dzucconi — APPROVED** [review]

File: src/Components/Inquiry/Views/InquirySpecialist.tsx (hunk 1 above)
Comment (MrSltun) [reply src/Components/Inquiry/Views/InquirySpecialist.tsx:62]: A toast is a great idea! I'll do that now :)

File: src/Components/Inquiry/config.ts [hunk 2]
```diff
@@ -31,14 +31,7 @@ export const useEngine = ({ context, onDone }: UseEngine) => {
...
           askSpecialist: {
-            true: [
-              "Specialist",
-              {
-                isLoggedOut: {
-                  true: ["Account"],
-                },
-              },
-            ],
+            true: ["Specialist"],
```
Comment (iskounen) [inline src/Components/Inquiry/config.ts:34]: suggestion (non-blocking): I understand the problem from the PR description, but I don't understand why this solves it. Perhaps a bit of context about why this solves the problem in the description would be helpful.

**Review by iskounen — APPROVED** [review]

File: src/Components/Inquiry/config.ts (hunk 2 above)
Comment (MrSltun) [reply src/Components/Inquiry/config.ts:34]: Great point! just updated it :)

**Pull request merged**